demo/server/Gruntfile.js

```diff
@@ -14,6 +14,7 @@ module.exports = function (grunt) {
         },
         exec: {
             'run': 'ts-node src/index.ts',
+            'runDebug': 'ts-node --inspect src/index.ts',
```

Comment:
hmm.. actually it's for life reload that was supported in previous application..
I couldn't really get that running so life reload will not work now :(